binding/form.go

```diff
@@ -52,5 +52,10 @@ func (formMultipartBinding) Bind(req *http.Request, obj interface{}) error {
 	if err := mapForm(obj, req.MultipartForm.Value); err != nil {
 		return err
 	}
+
+	if err := mapFiles(obj, req); err != nil {
```

Comment:
I hope `mapForm` and `mapFile` merged, please see the below example:
```go
type A struct {
  a string `form:"a" binding:"required"`
  b *multipart.FileHeader `form:"b" binding:"required"`
```
it will call `mapFiles` and `mapForm`, because skip `a` when call `mapFiles` and skip `b` when call `mapForm`.